media-sound/declick/declick-0.6.5-r1.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="declick is a dynamic digital declicker for audio sample files"
```

Comment:
no `PN` in descriptions. Just start with `Dynamic digital...`

Reply:
fixed :)